### PR TITLE
Hamming Code design changes

### PR DESCRIPTION
**Pull request by YouSafe**

### CODE REVIEW

**Comment by stefnotch** [comment]
Looks good to me, I'd be happy to merge this 🚀

File: src/pages/HammingCode.vue [hunk 1]
```diff
@@ -80,8 +131,9 @@ export default defineComponent({
     });
 
     const showCalulations = ref(false);
+    const showTable = ref(false);
 
-    const dataBitCalculation = ref(new DataBitCalculation(0));
+    const dataBitCalculation = ref(new DataBitCalculation(BigInt(0)));
```
Comment (Fancy11111) [inline src/pages/HammingCode.vue:136]: maybe we could use a urlRef here aswell?
Could clutter the url though, so I think it is fine either way

**Review by Fancy11111 — APPROVED** [review]
looks good to me

File: src/pages/HammingCode.vue (hunk 1 above)
Comment (stefnotch) [reply src/pages/HammingCode.vue:136]: I wonder, should I 
- add BigInt support to the urlRef
- or let the user pass a serializer/deserializer to the urlRef, sort of like `urlRef("data-bit-cat", v => BigInt(v), v => "" + v)`
- or add a fully blown arbitrary object serializer/deserializer

Comment (Fancy11111) [reply src/pages/HammingCode.vue:136]: i think for now we are good, but it definitely would be interesting for the future

**Pull request merged**